Indexer packages generation failing due to the alerting plugin

## Description

While analyzing https://github.com/wazuh/devel-major-incidents/issues/99, it has been detected that the Indexer packages are not being generated because of failures in the `publish-alerting` and `build-alerting-plugin` stages:

``` 
> Task :alerting:compileKotlin FAILED
e: file:///home/runner/work/wazuh-indexer/wazuh-indexer/alerting/src/main/kotlin/org/opensearch/alerting/resthandler/RestIndexMonitorAction.kt:105:17 'when' expression must be exhaustive. Add the 'ACTIVE_RESPONSE_MONITOR' branch or an 'else' branch.
gradle/actions: Writing build results to /home/runner/work/_temp/.gradle-actions/build-results/__run-1777945041395.json

[Incubating] Problems report is available at: file:///home/runner/work/wazuh-indexer/wazuh-indexer/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':alerting:compileKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details
```

## Evidence

- https://github.com/wazuh/wazuh-indexer/actions/runs/25352419448
- https://github.com/wazuh/wazuh-indexer/actions/runs/25352418755
- https://github.com/wazuh/wazuh-indexer/actions/runs/25352392786
- https://github.com/wazuh/wazuh-indexer/actions/runs/25352392253
- https://github.com/wazuh/wazuh-indexer/actions/runs/25351905816

--- a/.github/workflows/5_builderpackage_engine-standalone.yml
+++ b/.github/workflows/5_builderpackage_engine-standalone.yml
@@ -49,7 +49,7 @@ jobs:
     name: Build Manager (${{ matrix.arch }})
     runs-on:
       group: ${{ matrix.arch == 'arm64' && 'wz-linux-arm64' || 'wz-linux-amd64' }}
-      labels: ${{ matrix.arch == 'arm64' && 'wz-linux-arm64-2' || 'wz-linux-amd64' }}
+      labels: ${{ matrix.arch == 'arm64' && 'wz-linux-arm64' || 'wz-linux-amd64' }}
     outputs:
       version: ${{ steps.set_version.outputs.version }}
     strategy:
@@ -119,7 +119,7 @@ jobs:
 
     runs-on:
       group: ${{ matrix.arch == 'arm64' && 'wz-linux-arm64' || 'wz-linux-amd64' }}
-      labels: ${{ matrix.arch == 'arm64' && 'wz-linux-arm64-2' || 'wz-linux-amd64' }}
+      labels: ${{ matrix.arch == 'arm64' && 'wz-linux-arm64' || 'wz-linux-amd64' }}
     needs: build
     timeout-minutes: 60
     strategy:

